checklists/afd_checklist.en.json

```diff
@@ -175,6 +175,7 @@
             "guid": "2902d8cc-1b0c-4495-afad-624ab70f7bd6",
             "id": "A01.28",
             "severity": "High",
+            "graph" : "resources | where type == "microsoft.network/frontdoorwebapplicationfirewallpolicies" | where properties['policySettings']['mode'] == "Detection"| extend compliant = (properties['policySettings']['mode'] == "Detection")| distinct id, compliant",
```

Comment:
- Please use single quotes (instead of double quotes) inside of the ARG query string
- Please use the `=~` operator for comparisons